### ISSUE TITLE
send value to a component text an other page

### ISSUE THREAD

**Issue by anthology** [bborncr/nextion#38]
hello
it's possible to send a value of componentText on other page ?
this code work on the page where temp1 is but not on other page
`myNextion.setComponentText("temp1", String(temp1));
`
it's possible to send like this
`myNextion.setComponentText("page1.temp1", String(temp1));`
thanks a lot

**Comment by digi-thomas2003** [comment]
Yes it is possible: The vscope of the component must be set as _global_.

**Comment by anthology** [comment]
hello
thanks for your response, i have set global to all vscope of my component but when i send date, i received error

thanks a lot
